### ISSUE TITLE
Missing bundled fonts

### ISSUE THREAD

**Issue by SimenB** [unjs/jimp-compact#40]
```js
// file.js
const {existsSync} = require('fs')
const Jimp = require('jimp');
const JimpCompact = require('jimp-compact');

console.log(existsSync(Jimp.FONT_SANS_16_BLACK))
console.log(existsSync(JimpCompact.FONT_SANS_16_BLACK))
```

prints

```sh-session
$ node file.js
true
false
```

specifically, this fails for `Jimp.loadFont`

**Comment by pi0** [comment]
Thanks for reporting! Should be fixed in `0.16.1-1`.

**Comment by SimenB** [comment]
Works great, thanks!
